RPi4.md

```diff
@@ -187,7 +187,11 @@ This command allows SSH connections from your LAN only.<br/>
 **⚠️ Replace `192.168.1.0` with your own subnet:**
 
 ```bash
-sudo ufw allow from 192.168.1.0/24 to any port 22
+# local network
+ufw allow from 192.168.1.0/24 to any port 22
+
+# docker network, required for maintenance features
+ufw allow from 172.18.0.0/24 to any port 22
```

Comment:
@dennisreimann Did you test that this is enough and restarted the rpi several times and checked if Web UI SSH works still? I had different results, on some reboots it worked on some not. Did you find some information and have a link if Docker really only uses that subnet? 

I tried to whitelist the whole possible range of the 172.16. private IPs. But even adding that range I had non-consistent results and it did not work sometimes. There seems to be also that 169. range to be used? See https://github.com/btcpayserver/btcpayserver-docker/issues/198#issuecomment-534437298

Reply:
For me it worked when rebooting, didn't test more than three times though. But you are right and as the conversation in the issue you mentioned also states: In case the subnet of the `generated_default` Docker network changes this will break.

Any ideas on how we could handle this in a better way?

Reply:
```suggestion
sudo ufw allow from 10.0.0.0/8 to any port 22 proto tcp
sudo ufw allow from 172.16.0.0/12 to any port 22 proto tcp
sudo ufw allow from 192.168.0.0/16 to any port 22 proto tcp
sudo ufw allow from 169.254.0.0/16 to any port 22 proto tcp
sudo ufw allow from fc00::/7 to any port 22 proto tcp
sudo ufw allow from fe80::/10 to any port 22 proto tcp
sudo ufw allow from ff00::/8 to any port 22 proto tcp
```